### DISCUSSION TITLE
Haplotype storage in Breedbase

### DISCUSSION THREAD

**Discussion by lukasmueller** [solgenomics/sgn#3442] [Ideas]
Is it a good idea?

**Comment by bauchetg** [comment]
Hi Lukas in which way would it differ from the current marker storage?
Once a marker dataset is been phased and tagSNP (flanking markers delimiting a segment in LD) have been selected, we pretty much have an haplotype no?
If so, breedbase marker storage solution may already handle haplotype storage and would just require to create an haplotype specific genotyping/sequencing data upload protocol.
It could be useful is to include .hmp format as an alternate download output format than vcf.
https://bitbucket.org/tasseladmin/tassel-5-source/wiki/UserManual/Load/Load
https://bitbucket.org/bucklerlab/practicalhaplotypegraph/wiki/Home

**Comment by lukasmueller** [comment]
I think it would be very similar. The markers would just have many different states, identified by the haplotype block at that position. The haplotype blocks would be defined in the nd_protocol data structure. OK, I will now look at the links you sent...
